content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -24,27 +24,32 @@ to be enabled. The alternative is to delete the Pod and let the
 [workload controller](/docs/concepts/workloads/controllers/) make a replacement Pod
 that has a different resource requirement.
 
+A resize request is made through the pod `/resize` subresource, which takes the full updated pod for
+an update request, or a patch on the pod object for a patch request.
+
 For in-place resize of pod resources:
-- Container's resource `requests` and `limits` are _mutable_ for CPU
-  and memory resources.
-- `allocatedResources` field in `containerStatuses` of the Pod's status reflects
-  the resources allocated to the pod's containers.
-- `resources` field in `containerStatuses` of the Pod's status reflects the
-  actual resource `requests` and `limits` that are configured on the running
-  containers as reported by the container runtime.
-- `resize` field in the Pod's status shows the status of the last requested
+- A container's resource `requests` and `limits` are _mutable_ for CPU
+  and memory resources. These fields represent the _desired_ resources for the container.
+- The `resources` field in `containerStatuses` of the Pod's status reflects the resources
+  _allocated_ to the pod's containers. For running containers, this reflects the actual resource
+  `requests` and `limits` that are configured as reported by the container runtime. For non-running
+  containers, these are the resources allocated for the container when it starts.
+- The `resize` field in the Pod's status shows the status of the last requested
```

Comment:
Do you think, we should add a sentence when `resize` value is null ? For example if a pod was resized , and pending resize is null means that Kubelet has processed the resize ?